mac/expert/manual_install_troubleshooting.tw2

```diff
@@ -0,0 +1,12 @@
+::Manual Installation on a Mac - Expert - Troubleshooting
+# Installing Exercism on a Mac manually - Troubleshooting
```

Comment:
What do you suggest that we do differently for troubleshooting for experts here?

Reply:
Not right now-- the CLI is currently under development, so I don't have any particular steps to mention. How about we ditch this step until we actually have something to put in there?